### ISSUE TITLE
Add Backup All and Site Mount Terminus plugins to the directory

### ISSUE THREAD

**Issue by uberhacker** [pantheon-systems/documentation#2124]
## Issue Description
Please add Backup All and Site Mount Terminus plugins to directory.md.

## Suggested Resolution
Plugins appear in the directory in alphabetical order via #2123.

## Blocked By
- [ ] Quality standards for inclusion set by eng and documented in the Terminus guide (tracked internally

**Comment by alexfornuto** [comment]
Closing this in favor of a single issue to track all plugins awaiting entry: #2895
